src/js/background/context-menu.js

```diff
@@ -35,7 +35,7 @@ const staticMenuData = {
     title: browser.i18n.getMessage("pageInputIconInsertPhoneMask"),
     enabled: true,
     visible: true,
-    contexts: ["all"],
+    contexts: ["editable"],
```

Comment:
suggestion (non-blocking): I would recommend leaving this one as is. Having a way to open Relay anywhere can be useful. 

```suggestion
    contexts: ["all"],
```

Reply:
GitHub's diff collapsing makes this looks like this affects the `manageAliases` entry, but it's actually `insertPhoneMask`:

[Screencast from 2023-04-13 10-53-26.webm](https://user-images.githubusercontent.com/4251/231707777-67ec44e4-e4fa-498a-bc19-146006a59809.webm)

"Manage aliases" will indeed still be available everywhere.

(I was so confused when I went to make this change :sweat_smile: